invenio_rdm_records/fixtures/vocabularies.py

```diff
@@ -70,7 +71,7 @@ def map_row(self, header, row):
     def __iter__(self):
         """Iterate over records."""
         with open(self._data_file) as fp:
-            reader = csv.reader(fp, delimiter=";", quotechar='"')
+            reader = csv.reader(fp, delimiter=",", quotechar='"')
```

Comment:
minor: given that we're somewhat "breaking the API" for the CSV import, we could maybe use [the `csv.Sniffer`](https://docs.python.org/3/library/csv.html#csv.Sniffer), to detect the dialect (reading 1024 chars should be enough as done in the docs example).

Reply:
Good point, I'll add this here :+1: